web_timeline/static/src/js/timeline_view.js

```diff
@@ -84,8 +84,10 @@ odoo.define('web_timeline.TimelineView', function (require) {
             );
 
             this.parse_colors();
-            for (var i=0; i<this.colors.length; i++) {
-                fieldNames.push(this.colors[i].field);
+            for (let color of this.colors) {
```

Comment:
Odoo v11.0 doesn't support ES6. Can't use 'let' and 'of' :(
https://github.com/lukehoban/es6features#let--const

Reply:
Waiting for IE11 to be dropped :)